ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml

```diff
@@ -5437,7 +5453,7 @@
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-OXWSCORE_R1087</ns1:REQ_ID>
 		<ns1:Doc_Sect>2.2.4.27</ns1:Doc_Sect>
-		<ns1:Description>[In t:MeetingSuggestionType Complex Type] Location: An element of type string, as defined in [XMLSCHEMA2] section 3.2.1, that represents the location of the meeting.</ns1:Description>
+		<ns1:Description>[In t:MeetingSuggestionType Complex Type] Location: An element of type string, as defined in [XMLSCHEMA2] section 3.2.1, that represents the location of the meeting.&lt;88&gt;</ns1:Description>
 		<ns1:Behavior>Protocol</ns1:Behavior>

```

Comment:
It should be derived RS to verify like MS-OXWSCORE_R1730. Please add New RS and Comments.
Derived RS should be verified in Test Case.